modules/service/src/main/resources/app.conf

```diff
@@ -87,4 +87,11 @@ app {
     userinfo-endpoint  = "http://localhost:8080/userinfo"
     userinfo-endpoint  = ${?USERINFO_ENDPOINT}
   }
+
+  kafka {
+    permissions-projection-id = "permissions"
+    permissions-projection-id = ${?PERMISSIONS_PROJECTION_ID}
+    permissions-topic = "permissions"
```

Comment:
I'm wondering if we should also consider discriminating between the first level path, such that services can use this as some sort of view to avoid unnecessary noise, i.e.:
* permissions/kg: storing events that have the path `/kg/...`
* permissions/iam: storing events that have the path `/iam/...`
* permissions/serviceN: storing events that have the path `/serviceN/...`

Thoughts?

Reply:
yes, that make sense, especially that we'll add more services in the future.
@bogdanromanx in this case, where do we send permissions for `/`?

Reply:
Hmmm, good question... maybe to `permissions`, but that would imply having each service listen to two topics which is not that friendly. Anyone have other opinions?